doc/md/concepts/url.mdx

```diff
@@ -101,6 +101,14 @@ Connecting to an in-memory SQLite database (ephemeral). Useful for `--dev-url`:
 sqlite://file?mode=memory&_fk=1
 ```
 
+Atlas also supports WebSocket and HTTPS connections to remote `libsql` databases:
+
+```shell
+libsql+wss://database-url
+
+libsql+https://database-url
```

Comment:
https support is under development so it might be a good idea to keep it out of scope of this PR